src/UICrafter.Core/Components/Render/RenderAPIButton.razor

```diff
@@ -111,4 +125,32 @@
             inputFieldPublisher.Unsubscribe(pair.Key, UpdateURL);
         }
     }
+
+    private void DisplaySnackbar(string responseCode)
```

Comment:
Instead of doing this I think you should check IsSuccess and mark the severity based on that and then just read out the reason phrase from the http response, which will say Bad request or whatever.